build.sbt

```diff
@@ -558,6 +558,7 @@ lazy val partest = configureAsSubproject(project)
     name := "scala-partest",
     description := "Scala Compiler Testing Tool",
     libraryDependencies ++= List(testInterfaceDep, diffUtilsDep),
+    pomDependencyExclusions ++= List((organization.value, "scala-repl-frontend"), (organization.value, "scala-compiler-doc")),
```

Comment:
This is pretty fragile (I mean it should be defined with the `products` redefinition).

Though admittedly it should be a supported feature in sbt, as it's a common use case.